File: Easy Questions/Two_Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        cache = {}
        for n in range(len(nums)):
            num = nums[n]
            comp = target - num
            if comp in cache:
                return [cache[comp], n]
            cache[num] = n

File: Easy Questions/Two_Sum.go
func twoSum(nums []int, target int) []int {
    cache := make(map[int]int)
   for x, num := range nums{
    dif := target - num
    if  val, found := cache[dif]; found{
        return []int{x, val}
    }
    cache[num] = x;
   }
   return nil
}
